erpnext/selling/doctype/product_bundle/product_bundle.json

```diff
@@ -72,6 +75,37 @@
    "unique": 0
   }, 
   {
+   "allow_bulk_edit": 0, 
+   "allow_on_submit": 0, 
+   "bold": 0, 
+   "collapsible": 0, 
+   "columns": 0, 
+   "fieldname": "product_bundle_description", 
```

Comment:
how about just `description` this is the Product Bundle DocType, so it will be called as `product_bundle.description`